server.js

```diff
@@ -14,13 +14,32 @@
  * limitations under the License.
  */
 
+const isProd = Boolean(process.env.GAE_APPLICATION);
+
 const express = require('express');
-const app = express();
+const buildRedirectHandler = require('./redirect-handler.js');
 
-app.use(
+const handlers = [
   express.static('dist'),
   express.static('dist/en'),
-);
+];
+
+// In development, Eleventy isn't guaranteed to have run, so read the actual
+// source file.
+const redirectsPath = isProd
+    ? 'dist/en/_redirects.yaml'
+    : 'src/site/content/en/_redirects.yaml';
+
+// Don't block loading the server if the redirect handler couldn't build.
+try {
+  const redirectHandler = buildRedirectHandler(redirectsPath);
+  handlers.push(redirectHandler);
```

Comment:
nb. This could be `unshift` instead, if we think that this should win over actual content.

Reply:
I think push is the right behavior... If I recall the way devsite works is, if the file exists, it will use it. Otherwise it will redirect. This seems to match that behavior.